ts/src/okx.ts

```diff
@@ -1401,6 +1400,7 @@ export default class okx extends Exchange {
             'expiryDatetime': this.iso8601 (expiry),
             'strike': strikePrice,
             'optionType': optionType,
+            'created': this.safeInteger (market, 'listTime'),
```

Comment:
@ttodua can you add this to the market interface inside `types.ts` ?

Reply:
> @ttodua can you add this to the market interface inside `types.ts` ?

I don't think it should be added to type at this time, because it's (imo) experimental yet (i had a thread on discord, where I described it's misbehavior atm). so, maybe we can collect several topmost exchanges at first, add them in implementations and later "unify" them by exposing them in types and docs

Reply:
@ttodua I'm ok with that, I'm just not sure about the name, `created` seems too ambiguous (users might think it refers to the token creation and not the listing on the exchange)

Reply:
> @ttodua I'm ok with that, I'm just not sure about the name, `created` seems too ambiguous (users might think it refers to the token creation and not the listing on the exchange)

 I also said similar :) https://discord.com/channels/690203284119617602/1161255683992334379/1161256229159587872
but then as you read, i've changed mind back.

Reply:
sure no problem 🙂 , we can consider it the time when the market was created